### ISSUE TITLE
folder and files missing: python-numpy-devel

### ISSUE THREAD

**Issue by pulux** [voidlinux/void-packages#963]
The package miss some folders with all files:

numpy/core/include/numpy
numpy/f2py/src/

So it is not possible to compile software depends on.

**Comment by xtraeme** [comment]
```
[juan@leysa xbps]$ xbps-query -Rf python-numpy-devel
/usr/lib/python2.7/site-packages/numpy/random/randomkit.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/ufunc_api.txt
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/__ufunc_api.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/multiarray_api.txt
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/__multiarray_api.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/_numpyconfig.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/numpyconfig.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_3kcompat.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/utils.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_no_deprecated_api.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_cpu.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/arrayscalars.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/oldnumeric.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/arrayobject.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/noprefix.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_math.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_common.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_os.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/ufuncobject.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/halffloat.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/old_defines.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_endian.h
/usr/lib/python2.7/site-packages/numpy/core/include/numpy/npy_interrupt.h
/usr/include/python2.7/numpy -> /usr/lib/python2.7/site-packages/numpy/core/include/numpy
[juan@leysa xbps]$
```

**Comment by pulux** [comment]
Sorry only 
```
/usr/lib/python2.7/site-packages/numpy/f2py/src/fortranobject.c
/usr/lib/python2.7/site-packages/numpy/f2py/src/fortranobject.h
```
are missing.

**Comment by asergi** [comment]
Closed #963 via https://github.com/voidlinux/void-packages/commit/d05849a144cd31cf2481830ac86700dd2b9677f6.
